### ISSUE TITLE
Multiple issues with detection and formatting of SD card

### ISSUE THREAD

**Issue by cmdrogogov** [LordRafa/ALARMOnTegraK1#3]
Expected behavior;
Script formats SD card detected as /dev/mmbclk0 and proceeds to next steps

What happens;
Script fails to recognize SD card /dev/mmcblk0 as a valid target, fails out.

running lsblk -ndo tran gives three outputs;
```

 
sata
sata
```

If that SD detection block 

```
if [[ "$(lsblk -ndo tran ${1})" != "usb" ]]; then
  echo "Error: The selected device is not a USB/SD memory. Aborting to avoid damaging the system."
  exit
fi
```
is commented out;

Script then fails at make2fs with 

`The file /dev/mmblk01 does not exist and no size was specified`  even though the partition appears to have been created in the previous step.

Is the script expected to be run on a specific host platform in order to download and image the SDcard?  

I attempted on the latest arch 2020.10.01 liveUSB and an Ubuntu laptop I had laying around.

**Comment by cmdrogogov** [comment]
Update - 

Laptop internal SD reader vs. an external USB SD card adapter.   The script expects the SD card mounted on a USB adapter only.

**Comment by LordRafa** [comment]
I fixed this on commit: ca893d7e20bc3cc3c6e1ce7850c2c23ba215fe39, now it should be possible to use the internal reader. You will need to add the flag -f. This change will be released in a future release
